indica/workflows/bayes_workflow.py

```diff
@@ -441,38 +472,55 @@ def fit_ts_profile(self, data_context, split="LFS", quant="ne"):
             save_fig=True,
         )
         fit = xr.where(fit < 0, 1e-10, fit)
-        return fit
+        return fit, processed_data
 
-    def set_ts_profiles(self, data_context, split="LFS"):
+    def set_ts_profiles(self, data_context, split="LFS", R_shift=0.0):
 
-        ne_fit = self.fit_ts_profile(data_context, quant="ne", split=split) * 1e19
-        te_fit = self.fit_ts_profile(data_context, quant="te", split=split) * 1e3
+        ne_fit, ne_data = self.fit_ts_profile(
```

Comment:
It's probably worth adding a PPTS reader so that you can use the PPTS fits saved to MDS+. It's anyway the same fitting routine, and it's more consistent.